### DISCUSSION TITLE
Glitch

### DISCUSSION THREAD

**Discussion by FoundingTitanPrem** [aniyomiorg/aniyomi#319] [Q&A]
Every anime i search and open it , and see it's release date , it says "today" in all anime

**Comment by 3bodYking99** [comment]
Yeah, that's a problem I have too, not on all sites, but some if not most of them, Arabic, English, Spanish, and most languages, any site i see it always says "today"

**Reply by ahmedgamal17** [reply]
It's not a glitch these sites don't show episodes dates on their anime page
